lib/rialto/etl/cli/load.rb

```diff
@@ -19,6 +19,12 @@ class Load < Thor
         desc 'call NAME', "Call named loader (`#{@package_name} list` to see available names)"
         # Call a loader by name
         def call(name)
+          Dir["#{options[:input_file]}/*"].each do |file|
```

Comment:
What do you see as the advantage of this approach over, e.g.,:

```bash
$ find my/dir/full/of/sparql/ -type f -name \*.sparql -exec exe/load call Sparql -i {} \;
```

I ask because currently the `extract`, `transform`, and `load` CLIs operate on a single file, and we built the `grants` and `publications` CLIs to operate in bulk. That would change, making its responsibility somewhat less clear, and then we'd have to consider things like renaming the required `:input_file` arg to be clearer.

Reply:
Mostly my intent was that there is a `--skip_load` parameter on grants and publications, leaving a directory with A LOT of sparql files that need the manual `exe/load call Sparql` command. The `find` you provide here is certainly one option, and perhaps a better outcome here is to just document that, but I was going for something captured in the code so that the results of `--skip_load` is something usable with just the app and not something that requires a more complex command. But I see your point. I'm good either way really - especially considering writing a test for this condition is proving problematic within the existing test :-P

Reply:
I wonder if it would be better to add an `--input-dir` parameter instead of blurring the responsibility of `--input-file`...

Reply:
> Mostly my intent was that there is a --skip_load parameter on grants and publications, leaving a directory with A LOT of sparql files that need the manual exe/load call Sparql command. 

If you want to use `exe/grants` or `exe/publications` to load files without extracting or transforming, I believe you can invoke it without the `--force/-f` flag which will short-circuit the extract and transform operations. Will that work for you? Should we document that better?

Alternatively, I see that `exe/publications` has `--skip-extract` and `exe/grants` does not. We could add that flag to `exe/grants` and add `--skip-transform` to both of those. I think that might lead us to a cleaner and more consistent bunch o' CLI classes than expanding `exe/load` to the batch context.

Reply:
Oh yeah, that's a good solution, I didn't look at pubs and missed the `--skip-extract`... I'll update this. Thanks.